decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
Yes, the sei will be dropped by decodeNalu but the next NAL_SLICE_NONIDR will cause the V4L2_INPUT thread to crash. 
By applying this modification, yami can skip all the NONIDR until reach the next IDR for start playing. This shall be a better instead of thread crash, I think.